### PR TITLE
unistore: fix clustered index point range check.

### PR DESCRIPTION
**Pull request by coocood**
### What problem does this PR solve?

unistore executor incorrectly check range IsPoint, cause incorrect result.

### What is changed and how it works?

extract a method that also check that the number of column equals to the number of primaryCols for clustered index.

### Check List 

Tests 

- Unit test

### Release note 

- No release note

### CODE REVIEW

File: store/mockstore/unistore/cophandler/closure_exec.go [hunk 1]
```diff
@@ -389,6 +389,27 @@ func (e *closureExecutor) execute() ([]tipb.Chunk, error) {
 	return e.oldChunks, err
 }
 
+func (e *closureExecutor) isPointRange(ran kv.KeyRange) bool {
```
Comment (wjhuang2016) [inline store/mockstore/unistore/cophandler/closure_exec.go:392]: It's better to change the name to `isPointGet`

**Comment by coocood** [comment]
unrelated failure
```
[2020-06-18T13:07:39.138Z] ----------------------------------------------------------------------
[2020-06-18T13:07:39.138Z] FAIL: ddl_test.go:1122: testSuite6.TestSetDDLReorgBatchSize
[2020-06-18T13:07:39.138Z] 
[2020-06-18T13:07:39.138Z] ddl_test.go:1133:
[2020-06-18T13:07:39.138Z]     c.Assert(variable.GetDDLReorgBatchSize(), Equals, variable.MinDDLReorgBatchSize)
[2020-06-18T13:07:39.138Z] ... obtained int32 = 256
[2020-06-18T13:07:39.138Z] ... expected int32 = 32
```

**Pull request merged**